manifests/addons/dashboards/istio-performance-dashboard.json

```diff
@@ -119,13 +111,6 @@
           "intervalFactor": 1,
           "legendFormat": "istio-proxy",
           "refId": "C"
```

Comment:
@howardjohn after removing those deprecated stuff, should we update the `refId` to be `A` and `B`? since we only left two targets here.

Reply:
Maybe.. I am not sure the impact, do you know ifthere is any?

Reply:
by searching on https://grafana.com/docs/grafana/latest/packages_api/data/dataquery/#refid-property
you can see the refid is for querying purpose, if we do not have any query that uses/depends on refId, i think we'd better make it starts from `A` and so on so forth.